chart_review/agree.py

```diff
@@ -172,7 +200,7 @@ def csv_header(pick_label=False, as_string=False):
     :param pick_label: default= None
     :return: header
     """
-    as_list = ["F1", "Sens", "Spec", "PPV", "NPV", "TP", "FN", "TN", "FP"]
+    as_list = ["F1", "Sens", "Spec", "PPV", "NPV", "κ", "TP", "FN", "TN", "FP"]
```

Comment:
I'm being fancy here and above and using an actual unicode kappa character. I figured that was fine since this is used for presentation.

Reply:
If I wasn't on team @mikix I would fear the UNICODE but alas with you, I'm not panic.